third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -109,6 +109,10 @@ function handleItem(vm, gitContribute, gitUrlPattern) {
   if (vm.inheritance) {
     normalizeLanguageValuePairs(vm.inheritance).forEach(handleInheritance);
   }
+
+  if (vm.langs && vm.langs[0] === "python") {

```

Comment:
Please add a comment above this line with a little explanation.

Reply:
Also, is `vm.wrapcode` correct? Is all code wrapped, or just certain spots? `vm.wrapdeclarationcode` (:dizzy_face:)? Might not matter.

Reply:
Let's use `wrapdeclarationcode` :P

Added comment!